aspnetcore/migration/22-to-30.md

```diff
@@ -492,7 +492,11 @@ The default JSON serializer for ASP.NET Core is now <xref:System.Text.Json>, whi
                 new CamelCasePropertyNamesContractResolver());
   ```
 
-**Note:** If the `AddNewtonsoftJson` method is not available, make sure that you installed the [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson) package. A common error is to install the [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package instead of the [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson) package.
+**Note:** If the `AddNewtonsoftJson` method isn't available, make sure that you installed the [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson) package. A common error is to install the [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package instead of the [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson) package.
+
+### System.Runtime.Serialization attributes not supported
+
+The [[DataContract]](xref:System.Runtime.Serialization.DataContractAttribute) and [[DataMember]](xref:System.Runtime.Serialization.DataMemberAttribute) attributes aren't supported in `System.Text.Json`. For more information, see the GitHub issue [System.Text.Json support to System.Runtime.Serialization](https://github.com/dotnet/corefx/issues/38758).
```

Comment:
@scottaddie This is just one of many features that System.Text.Json doesn't have; this one and the others are all listed in the new Migrate from Newtonsoft.Json article that I added a link to at the start of this section. Adding this here could imply that this is the only issue we thought worth mentioning.

Reply:
Adding @Rick-Anderson, who added this text in https://github.com/aspnet/AspNetCore.Docs/pull/16537. Thoughts on what Tom stated above?

Reply:
@tdykstra can you fix that? Point to your article.

Reply:
👍